wiki/Tournaments/JOKE/1/en.md

```diff
@@ -0,0 +1,524 @@
+
+## Ruleset
+
+### Tournament Information
+
+1. Jaye's One Key Event (JOKE) is a 1v1 tournament; you're on your own out there!
+2. Beatmap scoring is based on ScoreV2.
+3. The maps for each round will be announced during a live showcase once all matches for a round have concluded (otherwise, it will be announced in the Discord server). Only the announced maps will be used during the respective matches.
+4. The tournament is Free Mod: players are free to use either Hidden, Fade In, Flashlight, Mirror (lol), or No Mod.
+5. Failed scores do not count towards the end result; in the event that both players fail, the higher scoring player will claim one point.
+6. If the map ends in a draw: The first draw - the map will be replayed. The second draw - the map will be nullified, no point(s) will be awarded, and the selector of the map may freely choose another map.
+7. If a player disconnects, they are treated as if they failed. Map aborting for a "false start" is up to the referee's discretion (and will likely take place if a disconnect occurs within the first 30 seconds of a map).
+8. Warmups are limited to 1-key Mania maps or 1-key Mania converts.
+9. The usual osu! rules apply; therefore, macroing/double-binding/and the likes, is strictly prohibited. If you want to be 100% safe, please just use one key (I mean, that is the entire point of the tournament).
+10. Don't be a douche to other players or members of staff; referees have permission to punish in accordance to your actions.
+
+### Registrations
+
+1. A registration form can be found on the Master Sheet.
+2. Participants are required to both complete the form and be a member of our Discord server. This is for identification/legitimacy of registration purposes and for convenience in posting updates about the tournament. If the registration form is filled out and the user is not present in the server when the Screening Phase concludes they will automatically be withdrawn from the registrant list.
+3. There are no restrictions for entry: any rank, any mode, and any country is allowed. Please note that osu! tournament staff hold the right to screen you out of the tournament.
+
+### Stage Instructions
+
+1. The first weekend will be the "Qualifiers". All participants will be assigned to a multiplayer lobby at a specified time. All participants will then play through the Qualifiers mappool, followed by a 10-15 minute break, after which they will play through the Qualifiers mappool for a second time.
+2. The bracket stage (following Qualifiers) is Double Elimination; the tournament will begin with a Split pool (meaning qualifying participants ranked 33-64 will be initially placed in Loser Bracket). This decision was made to knockout some weaker players earlier on (as this tournament is quite hard).
```

Comment:
```suggestion
2. The bracket stage (following Qualifiers) is Double Elimination — the tournament will begin with a Split pool (meaning qualifying participants ranked 33-64 will be initially placed in Loser Bracket). This decision was made to knock out some weaker players earlier on (as this tournament is quite hard).
```

Reply:
Would it be better if I moved the second half there to its own bullet point?